Arrowgene.Ddon.LoginServer/Handler/CreateCharacterHandler.cs

```diff
@@ -1131,6 +1131,51 @@ public override void Handle(LoginClient client, StructurePacket<C2LCreateCharact
                 }
             }
 
+            // Add current job's equipment to the equipment storage
+            List<Item?> performanceEquipItems = character.Equipment.GetEquipment(character.Job, EquipType.Performance);
+            for (int i = 0; i < performanceEquipItems.Count; i++)
+            {
+                Item? item = performanceEquipItems[i];
```

Comment:
What handles the item being null? Something in `setStorageItem`?

Reply:
passing null to setStorageItem is valid, it means setting that slot as free